meson.build

```diff
@@ -2,9 +2,11 @@ project('seldon', 'cpp',
   version : '0.1',
   default_options : ['warning_level=3', 'cpp_std=c++20', 'optimization=3'])
 
-add_global_arguments(['-Wno-unused-local-typedefs', '-Wno-array-bounds'], language : 'cpp')
+deps = []
+cpp_args = ['-Wno-unused-local-typedefs', '-Wno-array-bounds']
```

Comment:
It would be better to fix these than silence them..

Reply:
added it as compiler arguments in the _args list